Formula/mopidy.rb

```diff
@@ -1,25 +1,21 @@
 class Mopidy < Formula
   desc "Extensible music server written in Python"
   homepage "https://mopidy.com/"
-  url "https://files.pythonhosted.org/packages/c1/43/b450251b22c3b620a0f12bf7cb60db2ae160c5c535ce5dbe09c8ac35e1e3/Mopidy-3.4.1.tar.gz"
-  sha256 "5c047be65738924e0349357dc46cb159b7cfbe7ed8dd2c9f3ee750e30c7a459a"
+  url "https://files.pythonhosted.org/packages/cc/41/1f291572997c49fce9eef47cea6d06b7d30e9923cc75a84679767f7fc99e/Mopidy-3.4.2.tar.gz"
+  sha256 "ada9ecbfc09eecc8c9e6742a8a4fea1632a134a1ab060527d8aa3d36df0547b6"
   head "https://github.com/mopidy/mopidy.git"
 
-  depends_on "python@3.11"
-  depends_on "gst-plugins-base"
-  depends_on "gst-plugins-good"
-  depends_on "gst-plugins-bad"
-  depends_on "gst-plugins-ugly"
-  depends_on "gst-python"
+  depends_on "python@3.12"
+  depends_on "gstreamer"
```

Comment:
Replacing `gst-python` with `gstreamer` might be the right thing to do, but I haven't used Mac or Homebrew in a while, so I'm not familiar with any changes here.

Could you comment on the reasoning of this change?